docs/CheckList.md

```diff
@@ -0,0 +1,27 @@
+# Commit check list
+## Coding Conventions 
+### Django app
+- [ ] An app's name should be short, all-lowercase and not include numbers, dashes, periods, spaces, or special characters. It also, in general, should be the plural of an app's main model, so our  `posts`  app would have a main model called  `Post`.
```

Comment:
The PR is looking great, I would also add to avoid naming an app by a saved Django feature (like we first called our messaging app `messages` and it made a lot of problems :)